extensions/ql-vscode/src/abstract-webview.ts

```diff
@@ -35,20 +37,36 @@ export abstract class AbstractWebview<ToMessage extends WebviewMessage, FromMess
 
   public async restoreView(panel: WebviewPanel): Promise<void> {
     this.panel = panel;
-    this.setupPanel(panel);
+    const config = await this.getPanelConfig();
+    await this.setupPanel(panel, config);
```

Comment:
```suggestion
   this.setupPanel(panel, config);
```
It looks like `setupPanel` returns `void` rather than a promise.